code/game/machinery/autolathe/artist_bench.dm

```diff
@@ -62,7 +62,7 @@
 			if(H.stats.getPerk(PERK_ARTIST) && H.sanity.insight > 40)
 				ins_used = input("How much of your insight will you dedicate to this work? 40-[H.sanity.insight > 100 ? 100 : H.sanity.insight]","Insight Used") as null|num
 			else
-				ins_used = H.sanity.insight
+				ins_used = H.sanity.insight > 100 ? 100 : H.sanity.insight
```

Comment:
Maybe clamp function would fit better here? Ternary operator looks kinda messy.

Reply:
Factually incorrect, ternary operator is the best operator, and hardly worse than a function call.